### DISCUSSION TITLE
is not allowed Connection to China

### DISCUSSION THREAD

**Discussion by wyyw5701207** [rustdesk/rustdesk#7973] [Q&A]
I still used it yesterday, but today when I tried to connect, I was prompted that The connection is not allowed Connection to China is not allowed because of series scanning, please set up your own self hosted server
Is it necessary to build your own server starting today

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/7952

**Comment by fb2231** [comment]
我也没有搭建自己的服务器，有段时间没有用这个软件了，昨天打开软件建立连接的时候提示The connection is not allowed. Connection to China is not allowed because of serious scamming, please set up your own self-hosted server.目前还不知道是什么原因造成的
